### PR TITLE
Description of data flow

### LINKED ISSUE #68: Document the data flow in Notification Writer service
Document the data flow in Notification Writer service

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Description of data flow.

Fixes #68

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [x] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ Additionally this service exposes several metrics about consumed and processed
 messages. These metrics can be aggregated by Prometheus and displayed by
 Grafana tools.
 
+## Data flow
+
+1. Customer cluster with *Insights Operator* installed sends new data with info about cluster into *Ingress service*
+1. That service consumes such data, writes them into S3 Bucked and produce new message into Kafka topic named `platform.upload.buckit`.
```
Comment (epapbak) [inline docs/index.md:24]: S3 bucket

File: docs/index.md [hunk 2]
```diff
@@ -18,6 +18,18 @@ Additionally this service exposes several metrics about consumed and processed
...
+1. Event about new data sent by *Insights Operator* is consumed from Kafka topic `platform.upload.buckit` by *CCX Data pipeline* service.
+1. That event contains (among other things) an URL to S3 Bucket.
+1. Insights operator data is read from S3 Bucket and *insights rules* are applied to that data in `ccx-data-pipeline` service.
+1.  Results (basically `organization ID` + `cluster name` + `insights results JSON`) are stored back into Kafka, but into different topic named `ccx.ocp.results`.
```
Comment (epapbak) [inline docs/index.md:28]: Since we're at it, please remove the extra whitespace between "1." and "Results"

File: docs/index.md [hunk 3]
```diff
@@ -18,6 +18,18 @@ Additionally this service exposes several metrics about consumed and processed
...
+1. That results are consumed by `ccx-notification-writer` service.
+1. `ccx-notification-writer` servicestores insights results into AWS RDS database into `new_reports` table.
```
Comment (epapbak) [inline docs/index.md:30]: service stores

**Review by epapbak — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (tisnik) [reply docs/index.md]: fixed, thank you

**Pull request merged**